app/@nadya:v1/LOG

```diff
@@ -0,0 +1,3 @@
+2019/06/26-18:47:03.110900 70000f678000 Recovering log #15
```

Comment:
I never expect this files before. where did they came?

Reply:
I remove directory , that’s directory has created if multiple git remote on the same url 

origin: xxx.git
upstream: xxx.git 

I think, I’m still working on forked repo but actually in same repostory. :sorry for my bad

Reply:
Yeaa no problem